pyproject.toml

```diff
@@ -10,10 +10,10 @@ homepage = "https://github.com/AlertaDengue/epi-scanner-downloader"
 packages = [{include="scanner", from="src"}]
 
 [tool.poetry.dependencies]
-python = ">=3.10,<3.12"
+python = ">=3.9,<3.11"
```

Comment:
If it is really necessary to include 3.9; maybe add 3.11 here, and then we would have the last 3 versions of Python. So in this case it would be something like: ``` ">=3.9,<3.12" ``` And here we would have the [3.9, 3.10, 3.11]

Reply:
In this case, it would require to be changed in the AD's repo as well.. AD won't allow the py version 3.11 in its deps

Reply:
I'm removing some unnecessary dependencies on poetry packages, do you want me to open a specific PR for that?